src/aat/resources/features/F-051 - Get Profile/S-109.td.json

```diff
@@ -343,7 +343,9 @@
 								{
                   "_extends_": "CaseworkerCaaAccessControlList"
 								}
-							]
+							],
+              "callbackGetCaseUrl": null,
+              "retriesGetCaseUrl": null
```

Comment:
this one has been changed to List<Integer> in definition store

Reply:
done